File: algorithm/src/main/java/baekjoon/p11006/Main.java
package baekjoon.p11006;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();

        while (t-- > 0) {
            for (int solve : solve(sc.nextInt(), sc.nextInt())) {
                System.out.print(solve + " ");
            }
            System.out.println();
        }
    }

    private static int[] solve(int n, int m) {
        return new int[]{
                m * 2 - n,
                m - (m * 2 - n)
        };
    }

}

File: algorithm/src/main/java/baekjoon/p11006/Main.kt
package baekjoon.p11006

fun main(args: Array<String>) {
    repeat(readLine()!!.toInt()) {
        val inputs = readLine()!!.split(" ").map { it.toInt() }
        println(solve(inputs[0], inputs[1]).joinToString(" "))
    }
}

private fun solve(n: Int, m: Int): IntArray {
    return intArrayOf(
            m * 2 - n,
            m - (m * 2 - n)
    )
}